xwalk_android.gypi

```diff
@@ -213,6 +213,59 @@
       'includes': ['../build/jni_generator.gypi'],
     },
     {
+      'target_name': 'xwalk_runtime_lib_lzma_compress',
+      'type': 'none',
+      'dependencies': [
+        'xwalk_runtime_lib_apk',
+      ],
+      'variables': {
+        'files_to_compress': [
+          '<(PRODUCT_DIR)/xwalk_runtime_lib_apk/stripped_libraries/libxwalkcore.so',
+          '<(PRODUCT_DIR)/xwalk_runtime_lib_apk/classes.dex',
+          '<(PRODUCT_DIR)/xwalk_runtime_lib/assets/icudtl.dat',
+          '<(PRODUCT_DIR)/xwalk_runtime_lib/assets/xwalk.pak',
+        ],
+        'assets_dir': '<(PRODUCT_DIR)/xwalk_runtime_lib_lzma/assets/',
+      },
+      'actions': [
+        {
+          'action_name': 'lzma_compress',
+          'message': 'compress ingredients for runtime APK',
+          'inputs': [
+            'build/android/lzma_compress.py',
+          ],
+          'outputs': [
```

Comment:
> I tried ninja -t clean and it works.

All files in `out/Release/xwalk_runtime_lib_lzma/assets/` remain there after the call to `-t clean`, the only file that gets removed is the stamp.

> Actually, there's no any target directly depends on the files generated here, the action here is just used to prepare assets for XWalkRuntimeLibLzma APK generation.

I understand that, yet time and again we've been bitten by this: a year from now someone copy-pastes this block elsewhere to implement a new feature, or this doesn't work as expected with GN, or someone uses this target in a new one and things work unpredictably. I really don't see any downside to properly specifying all your outputs.